src/bci_build/package/golang/README.md.j2

```diff
@@ -58,26 +75,44 @@ To run the application tests inside a container, use the following command:
 $ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} go test -v
 ```
 
-**Note:** The Golang image should be used as a build environment. For runtime, self-contained Go binaries should use a `scratch` image and for applications that require external dependencies use the `bci-base` image.
+**Note:** The Go image should be used as a build environment. That means
+the compiler does not need to be shipped as part of the images that are
+distributed. Instead, it is recommended that the Go image is used as the
+builder image only.
 
-## Additional tools
+There are two ways to work with Go images. First, you can encapsulate your
+application in a `scratch` container image, which is essentially an empty
+image. This approach will not function if your Go application depends on libc
+or any other library, as they will not be available.
```

Comment:
```suggestion
or any other library, as the libraries will not be available.
```

Reply:
I have literally copied this text from https://opensource.suse.com/bci-docs/guides/use-with-golang/ - can we update it there?